gensrc/proto/internal_service.proto

```diff
@@ -116,6 +116,12 @@ message PExecPlanFragmentResult {
 
 message PCancelPlanFragmentRequest {
     required PUniqueId finst_id = 1;
+    enum CancelReason {
```

Comment:
I think it's better to move CancelReason out of `PCancelPlanFragmentRequest`. Because fragment can be cancelled in other format. For example when Fragment is timed out, cancel thread will cancel this instance.

Reply:
ok